src/package-version.tsx

```diff
@@ -1,3 +1,3 @@
-import {version} from '../package.json';
+import pkg from '../package.json';
```

Comment:
new playroom version uses webpack 5 and was causing this warning: https://jira.tid.es/browse/WEB-214

Reply:
can we use babel macro here?

Reply:
we don't use babel, so no babel macro. I don't know if there is something similar for swc.
We could write a file with the version as part of the release workflow

Reply:
this should do the trick: [`943bd12` (#384)](https://github.com/Telefonica/mistica-web/pull/384/commits/943bd12b2b5e7ce77dbb74817a954d2de599783c)